apm-sniffer/apm-sdk-plugin/httpClient-4.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/httpClient/v4/HttpClientExecuteInterceptor.java

```diff
@@ -69,7 +60,7 @@
 
     @Override public Object afterMethod(EnhancedInstance objInst, Method method, Object[] allArguments,
         Class<?>[] argumentsTypes, Object ret) throws Throwable {
-        if (allArguments[0] == null || allArguments[1] == null) {
+        if (allArguments[0] == null || allArguments[1] == null || ret == null) {
```

Comment:
You can't just simply return. This will cause a leak of context. In `before`, span created. So you need to close it before return.

I understood you didn't want to tag for avoiding to read `ret`, but please do `stopSpan` at least.